shared/AppInsightsCommon/tslint.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
Can you have a single tslint.json used by all packages?

Reply:
@markwolff I don't think so?

Reply:
You can have a `tslint.base.json` at the root dir which the tslint of each package would `"extends": "../../tslint.base.json"` on